src/network/networkd-network.c

```diff
@@ -1286,13 +1287,15 @@ int config_parse_dhcp_route_table(const char *unit,
                                   const char *rvalue,
                                   void *data,
                                   void *userdata) {
+        Network *network = userdata;
```

Comment:
hmm, so this works, but I think it's a bit misleading now, since we don't use data at all anymore, but the gperf table offset still lists something not 0 as offset. I think it would be nicer to change the gperf table to use offset 0 (which means data and userdata point to the same location) and then use `data` here, as before, and not userdata.

Reply:
Ahh, that makes sense. I wanted to look into how the gperf stuff actually works. Thanks for explaning that part 👍 
I'll change accordingly.